src/site/content/en/blog/lighthouse-whats-new-6.0/index.md

```diff
@@ -0,0 +1,463 @@
+---
+title: "What's New in Lighthouse 6.0"
+authors:
+  - cjamcl
+date: 2020-05-18
+hero: lighthouse-logo.svg
+alt: Lighthouse logo.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - fast
+  - lighthouse
+---
+
+Today we're releasing Lighthouse 6.0!
+
+[Lighthouse](https://github.com/GoogleChrome/lighthouse/) is an automated website auditing tool that
+helps developers with opportunities and diagnostics to improve the user experience of their sites.
+It's available in Chrome DevTools, npm (as a Node module and a CLI), or as a browser extension (in
+[Chrome](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk) and
+[Firefox](https://addons.mozilla.org/en-US/firefox/addon/google-lighthouse/)). It powers many Google
+services, including [web.dev/measure](https://web.dev/measure/) and [PageSpeed
+Insights](https://developers.google.com/speed/pagespeed/insights/).
+
+Lighthouse 6.0 is available immediately on npm and in [Chrome
+Canary](https://www.google.com/chrome/canary/). Other Google services that leverage Lighthouse will
+update by the end of the month. It will land in Chrome Stable in Chrome 84 (mid-July).
+
+To try the Lighthouse Node CLI, use the following commands:  
+```bash
+npm install -g lighthouse
+lighthouse https://www.example.com --view
+```
+
+This version of Lighthouse comes with a large number of changes that are all listed in the
+changelog. (**TODO: link to changelog)**. We'll cover the highlights in this article.
+
+- [New metrics](#new-metrics)
+- [Performance score update](#score)
+- [New audits](#new-audits)
+- [Lighthouse CI](#ci)
+- [Renamed Chrome DevTools panel](#devtools)
+- [Mobile emulation](#emulation)
+- [Browser extension](#extension)
+- [Budgets](#budgets)
+- [Source location links](#source-location)
+- [On the horizon](#horizon)
+- [Thank you!](#thanks)
+
+## New metrics {: #new-metrics }
+
+![image](lighthouse-new-metrics.png)
+
+Lighthouse 6.0 introduces three new metrics to the report. Two of these new metrics–Largest
+Contentful Paint (LCP) and Cumulative Layout Shift (CLS)–are lab implementations of [Core Web
+Vitals](https://web.dev/vitals/).
+
+### Largest Contentful Paint (LCP) {: #lcp }
+
+[Largest Contentful Paint (LCP)](https://www.web.dev/lcp/) is a measurement of perceived loading
+experience. It marks the point during page load when the primary–or "largest"–content has loaded and
+is visible to the user. LCP is an important complement to First Contentful Paint (FCP) which only
+captures the very beginning of the loading experience. LCP provides a signal to developers about how
+quickly a user is actually able to see the content of a page. An LCP score below 2.5 seconds is
+considered 'Good.'
+
+For more information, watch this deep-dive on LCP by Paul Irish:
+
+{% YouTube 'diAc65p15ag' %}
```

Comment:
I don't know about others but I dont love the YT embed

also it doesnt use youtube-lite, which i added to web.dev in a PR in november. 😭 

personally i'd prefer these sit as text links.

Reply:
> also it doesnt use youtube-lite, which i added to web.dev in a PR in november. 😭

[it was all smooshed together!](https://github.com/GoogleChrome/web.dev/pull/1882/files#diff-87fd67e4e0f3e4c45fd06774bbcdaa8eR294)

I think we'd need someone to do a pass to clean it up and check it in but it's been hard to get it prioritized with web.dev/live and other feature requests coming in 😅 I'll take a note to see if I can get it on the board in case folks have a spare moment in the coming weeks/months.